registry/src/skills.json

```
@@ -7,6 +7,92 @@
         "author": null,
         "description": "Enables your agent to fetch token details for any token on Base",
         "githubUrl": "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-moxie-token-details",
+        "imageUrl": "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-moxie-token-details/images/logo.png"
+    },
+    {
+        "pluginId": "ba2e0746-d089-4539-a23b-84882bc8eafd",
+        "name": "@moxie-protocol/plugin-betswirl",
+        "version": "0.0.1",
+        "displayName": "Play Games of Luck",
+        "author": "Romuald Hog <romuald@betswirl.com>",
+        "description": "Enabled your agent to place bets on BetSwirl's games of luck, and retrieving bets information.",
+        "githubUrl": "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-betswirl",
+        "logoUrl": "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-betswirl/images/logo.png",
```

changing path, for image this should be correct one
```suggestion
        "logoUrl": "https://raw.githubusercontent.com/moxie-protocol/moxie-agent-skills/refs/heads/main/packages/plugin-betswirl/images/logo.png",
```